src/inmanta/env.py

```diff
@@ -534,13 +484,12 @@ def run_pip_install_command_from_config(
             # If we don't use system config, unset env vars
             # setting this env_var to os.devnull disables the loading of all pip configuration files
 
+            # Drop all PIP env vars
+            for env_var in list(sub_env.keys()):
+                if env_var.startswith("PIP_"):
```

Comment:
I think this is a bit too strict. I thought we concluded we would only explicitly ignore the env vars that we know about?

From the design: "other pip-related environment variables are not overridden/ignored because inmanta is not aware of them."

I am open to discussion on this point, as I think there is something to say for both approaches. Mainly, ignoring all of them eases backwards compatibility when adding support for new options in the future. But on the other hand there is the fact that in this case any option that we don't support would require `use-system-config=true`, which I don't really like.

Reply:
Leaving this unresolved because I linked to it from another PR, and GitHub doesn't like links to resolved conversations.